docs/optimization_guide/dldt_deployment_optimization_tput_advanced.md

```diff
@@ -1,77 +1,81 @@
-Run performance testing in the scope of development, and make sure to validate overall (end-to-end) application performance.
+Run performance testing in the scope of development, and make sure to validate overall (*end-to-end*) application performance.
 
 Different devices behave differently with the batch sizes. The optimal batch size depends on the model, inference precision and other factors.
 Similarly, different devices require different number of execution streams to saturate.
-Finally, in some cases  combination of streams and batching may be required to maximize the throughput.
+In some cases  combination of streams and batching may be required to maximize the throughput.
 
 One possible throughput optimization strategy is to **set an upper bound for latency and then increase the batch size and/or number of the streams until that tail latency is met (or the throughput is not growing anymore)**.
 Also, consider [OpenVINO Deep Learning Workbench](@ref workbench_docs_Workbench_DG_Introduction) that builds handy latency vs throughput charts, iterating over possible values of the batch size and number of streams.
 
 > **NOTE**: When playing with [dynamically-shaped inputs](../OV_Runtime_UG/ov_dynamic_shapes.md) use only the streams (no batching), as they tolerate individual requests having different shapes. 
 
-> **NOTE**: Using the [High-Level Performance Hints](../OV_Runtime_UG/performance_hints.md) is the alternative,  portable and future-proof option, allowing the OpenVINO to find best combination of streams and batching for a given scenario and model. 
+> **NOTE**: Using the [High-Level Performance Hints](../OV_Runtime_UG/performance_hints.md) is the alternative, portable and future-proof option, allowing the OpenVINO to find best combination of streams and batching for a given scenario and model. 
 
+@anchor stream_considerations
 ### Number of Streams Considerations
-* Select the number of streams is it is **less or equal** to the number of requests that your application would be able to runs simultaneously
-* To avoid wasting resources, the number of streams should be enough to meet the _average_ parallel slack rather than the peak load
-* As a more portable option (that also respects the underlying hardware configuration) use the `ov::streams::AUTO`
-* It is very important to keep these streams busy, by running as many inference requests as possible (e.g. start the newly-arrived inputs immediately)
-   * Bare minimum of requests to saturate the device can be queried as `ov::optimal_number_of_infer_requests` of the  `ov:Compiled_Model` 
-* _Maximum number of streams_ for the device (per model) can be queried as the `ov::range_for_streams`
+* Select the number of streams that is **less or equal** to the number of requests that application would be able to runs simultaneously.
+* To avoid wasting resources, the number of streams should be enough to meet the _average_ parallel slack rather than the peak load.
```

Comment:
```suggestion
* To avoid wasting resources, the number of streams should be enough to meet the *average* parallel slack rather than the peak load.
```